#! /bin/sh -e

# All lines beginning with `# DP:' are a description of the patch.
# DP: Description: #327025 C locale has bad default for _NL_TIME_FIRST_WEEKDAY
# DP:    Also fixes the 3rd value of the week keyword.
# DP: Related bugs: #327025  BZ181
# DP: Dpatch author: Denis Barbier
# DP: Patch author: Petter Reinholdtsen
# DP: Upstream status: BZ181
# DP: Date: 2006-01-10

Index: locale/C-time.c
===================================================================
--- locale/C-time.c.orig	2007-10-07 16:35:43.000000000 -0400
+++ locale/C-time.c	2007-10-22 22:16:39.000000000 -0400
@@ -135,10 +135,10 @@
     { .wstr = (const uint32_t *) L"" },
     { .string = "\7" },
     { .word = 19971201 },
-    { .string = "\4" },
     { .string = "\7" },
     { .string = "\1" },
     { .string = "\1" },
+    { .string = "\1" },
     { .string = "" },
     { .string = "%a %b %e %H:%M:%S %Z %Y" },
     { .wstr = (const uint32_t *) L"%a %b %e %H:%M:%S %Z %Y" },
